src/javascripts/jquery.tipsy.js

```diff
@@ -40,6 +40,7 @@
                 return;
             }
 
+            var parent = this.options.appendToElement ? this.$element[0] : document.body;
```

Comment:
How about this?

```suggestion
            var $parent = this.options.appendToElement ? this.$element : document.body;
```

We don't need to extract the DOM element from the jQuery wrapper